static/app/views/insights/mobile/screenload/views/screenLoadSpansPage.tsx

```diff
@@ -67,6 +67,9 @@ function ScreenLoadSpans() {
     spanDescription,
   } = location.query;
 
+  const countIfPrimaryRelease = `count_if(measurements.time_to_initial_display,release,${primaryRelease})`;
+  const countIfSecondaryRelease = `count_if(measurements.time_to_initial_display,release,${secondaryRelease})`;
```

Comment:
Personally I'd probably keep these inlined so a reader doesn't have to jump around but that's a tiny nitpick.

Reply:
I had to change it a few times in two places, but either we have variables for all of these or none. I change it back to having it duplicated.